CMakeLists.txt

```diff
@@ -314,34 +314,28 @@ endif (NOT MACOSX)
 find_package (OpenSSL REQUIRED)
 set (INCLUDE_DIRECTORIES ${INCLUDE_DIRECTORIES} ${OPENSSL_INCLUDE_DIR})
 
-find_package (GTest REQUIRED)
-set (INCLUDE_DIRECTORIES ${INCLUDE_DIRECTORIES} ${GTEST_INCLUDE_DIRS})
+if (BUILD_TESTING OR BUILD_UNITTESTS OR BUILD_UNITTESTS_DEBUG)
+  find_package (GTest REQUIRED)
+  set (INCLUDE_DIRECTORIES ${INCLUDE_DIRECTORIES} ${GTEST_INCLUDE_DIRS})
 
-find_package (GMock REQUIRED)
-set (INCLUDE_DIRECTORIES ${INCLUDE_DIRECTORIES} ${GMOCK_INCLUDE_DIRS})
+  find_package (GMock REQUIRED)
+  set (INCLUDE_DIRECTORIES ${INCLUDE_DIRECTORIES} ${GMOCK_INCLUDE_DIRS})
+endif(BUILD_TESTING OR BUILD_UNITTESTS OR BUILD_UNITTESTS_DEBUG)
 
-find_package (VJSON REQUIRED)
+add_subdirectory(externals/vjson)
```

Comment:
I guess that the headers and static libraries are not ending up anymore in the externals directory.  Where will they end up?

Reply:
They end up in `${CMAKE_BINARY_DIR}/externals/vjson` during the build (i.e. not in `${CMAKE_SOURCE_DIR}/externals_build/vjson`).